### ISSUE TITLE
insufficient inputs

### ISSUE THREAD

**Issue by mhrimaz** [se2p/programming-styles-ss-2020-program-description#15]
> In case there are no more input to process, but the game is not yet finished, the GUI must be cleared and the following error message must be shown (i.e., no board, just the error message).

I case that inputs are not enough to reach a final game state, should we show just one error message, or should we print the boards until the final step and then print that error message?

for example if the input is `[1]` how the output should looks like?
```
┌─────────────────┐
│ Missing inputs. │
│ The games ends! │
└─────────────────┘
```
Or
```
╔════════╤═══╤═══╤═══╤═══╤═══╤═══╤═══╤═══╤═══╤═══╤═══════╗
║  ┌──┐1 │   │   │   │   │ 2 │   │   │   │   │   │       ║
║  │  │  │   │   │   │   │ 3 │   │   │   │   │   │       ║
║  │  │  │   │   │   │   │ 4 │   │   │   │   │   │       ║
║  └──┘6 │   │   │   │   │ 5 │   │   │   │   │   │       ║
╚════════╧═══╧═══╧═══╧═══╧═══╧═══╧═══╧═══╧═══╧═══╧═══════╝
┌─────────────────┐
│ Missing inputs. │
│ The games ends! │
└─────────────────┘
```

**Comment by alessiogambi** [comment]
Good question. The intended behavior is the second one. However, now that I see it, maybe for the sake of consistency, it might be better to show the banner just as we do in the other cases. What do you think?

According to this, your example should look like (this only represents the LAST screen shown):

```
 ╔════════╤═══╤═══╤═══╤═══╤═══╤═══╤═══╤═══╤═══╤═══╤═══════╗
┌──────────────────────────────────────────────────────────┐
│                      Missing inputs.                     │
│                      The games ends!                     │
└──────────────────────────────────────────────────────────┘
 ╚════════╧═══╧═══╧═══╧═══╧═══╧═══╧═══╧═══╧═══╧═══╧═══════╝
```

**Comment by alessiogambi** [comment]
This has been solved by #18
